R/get-param.R

```diff
@@ -0,0 +1,128 @@
+#' Return theta values as a named vector
+#'
+#' @param .summary  A `bbi_nonmem_summary` or `bbi_summary_list` object.
+#' @export
+get_theta <- function(.summary){
+
+  if (inherits(.summary, "bbi_nonmem_summary")) {
```

Comment:
Seems a bit odd to use "bbi_nonmem_summary" directly here and then the `NM_SUM_CLASS` indirection in the `check_model_object_list` right below.

Reply:
We have similar code [here](https://github.com/metrumresearchgroup/bbr/blob/main/R/test-threads.R#L180-L184), but I take your point. Its interesting we dont seem to have one for `"bbi_summary_list"`
```r
> NM_SUM_CLASS_LIST
[1] "bbi_nonmem_summary" "bbi_model"          "list"    
```